File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/library/permissions/storage/request/write/IStorageWritePermissionsRequestNotificationBuilder.java
package com.lasthopesoftware.bluewater.client.library.permissions.storage.request.write;

import android.app.Notification;

/**
 * Created by david on 7/3/16.
 */
public interface IStorageWritePermissionsRequestNotificationBuilder {
	Notification buildWritePermissionsRequestNotification(int libraryId);
}

File: projectBlueWater/src/main/java/com/lasthopesoftware/bluewater/client/stored/library/permissions/write/IStorageWritePermissionsRequestNotificationBuilder.kt
package com.lasthopesoftware.bluewater.client.stored.library.permissions.write

import android.app.Notification

/**
 * Created by david on 7/3/16.
 */
interface IStorageWritePermissionsRequestNotificationBuilder {
    fun buildWritePermissionsRequestNotification(libraryId: Int): Notification?
}
